Modules/_xxtestfuzz/fuzzer.c

```diff
@@ -439,6 +439,68 @@ static int fuzz_ast_literal_eval(const char* data, size_t size) {
     return 0;
 }
 
+#define MAX_ELEMENTTREE_PARSEWHOLE_TEST_SIZE 0x100000
+PyObject* xml_parser_type = NULL;
+PyObject* bytesio_type = NULL;
+/* Called by LLVMFuzzerTestOneInput for initialization */
+static int init_elementtree_parsewhole(void) {
+    PyObject* elementtree_module = PyImport_ImportModule("_elementtree");
```

Comment:
Alternative to my other comments... simplify this by using our internal `_PyImport_GetModuleAttrString` API.

Reply:
I went with the explicit refcounting fix instead.